code/_core/datum/experience/skills.dm

```diff
@@ -140,4 +140,13 @@
 	desc = "This doesn't work, yet!"
 	desc_extended = "Your skill in enchanting objects. Affects the speed, sucess rate, and quality of enchanted items."
 	experience_power = 1.8
-	experience_multiplier = 4.1
\ No newline at end of file
+	experience_multiplier = 4.1
+
+/experience/skill/botany/ //Int
```

Comment:
I know that this is gonna sound more of a "me" thing, but I really think that botany should use your strength or endurance-like stat, considering how much work do all of the farmers have to put into crops.

Reply:
Upon digging, this was an old comment from the previous iteration. As it exists in the current code, Botany skill doesn't actually rely on any specific stat. Updating comments to reflect this.